scripts/docs.js

```diff
@@ -0,0 +1,274 @@
+/**
+ * Copyright 2014 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var globby = require('globby');
+var async = require('async');
+var fs = require('fs');
+var dox = require('dox');
+var path = require('path');
+var exec = require('propprop');
```

Comment:
We usually call this `prop`. `exec` is for `methmeth`. The difference is `prop` just returns the property, where `exec` will execute the method.